src/patterns/molecules/breadcrumb/breadcrumb.hbs

```diff
@@ -0,0 +1,21 @@
+---
+name: Breadcrumb
+description: A type of navigation menu used to provide a list of links based on current page within the site hierarchy, allowing users to visit previous pages from the current page.
+order: 1
+notes: |
+    - Use the `.mzp-t-dark` options to use dark mode.
+    - Use a `<ol>` to use semantic HTML and structure.
+    - Using the `nav` element is optional, but reccomended.
+---
+<nav aria-label="breadcrumb">
```

Comment:
I also [commented elsewhere](https://github.com/mozilla/protocol/pull/730#issuecomment-961258254) but repeating here so it's not lost: after a little research I think we should indeed require breadcrumbs to be in a `nav` and make this `<nav class="mzp-c-breadcrumb" aria-label="breadcrumb">`